### ISSUE TITLE
[KT] (AICCRA) Show information in tabs for Progress and Reporting phases in Contribution to Performance indicator Cluster Section 

### ISSUE THREAD

**Issue by kenjitm** [CCAFS/MARLO#2343]
In AR 2021 phase, show in tabs the information of progress 2021 and of AR 2021

![image.png](https://images.zenhubusercontent.com/5b743d40a6afdd0ad0617092/1cc75b91-bf85-4cd1-bb49-34a12cc5f17f)

**Comment by kenjitm** [comment]
@MargaritaRamirez The information is visible in tabs for each phase (ready in AICCRA dev)
![image.png](https://images.zenhubusercontent.com/5b743d40a6afdd0ad0617092/cb411ab1-9a21-430c-a829-95aaf7b3e5b6)

It's pending to define:
- [ ] The replication of the guidance document 
- [ ] The process of organizing the previous information 
(these doubts will be clarified in a meeting held tomorrow)

**Comment by kenjitm** [comment]
![image.png](https://images.zenhubusercontent.com/5b743d40a6afdd0ad0617092/af988448-96e7-4c80-b21e-81b1f0854741)
- [ ] The progress to key performance indicators are displayed per phase in tabs in AR phases
- [ ] In AR Phase, the Progress information is not editable
- [ ] The questions are unique by phase
- [ ] The information that has been replicated to subsequent phases before carrying out the development
 was fixed at the database level

**Comment by MargaritaRamirez** [comment]
@htobon do we need to remove this?
![image.png](https://images.zenhubusercontent.com/5c6c4b4e3ba63a679b1c58cc/24ec909f-a304-4da0-9ff1-37246252203c)

**Comment by MargaritaRamirez** [comment]
Made some test and is working fine.

**Comment by htobon** [comment]
Yes, please go ahead and take out that question related to communications.

**Comment by kenjitm** [comment]
@MargaritaRamirez @htobon It was found that the question related to communications is a specificity, it is deactivated in test and production

**Comment by kenjitm** [comment]
Some labels were updated for AR Phase and the _Reporting 2021_ tab is selected by default

**Comment by MargaritaRamirez** [comment]
![image.png](https://images.zenhubusercontent.com/5c6c4b4e3ba63a679b1c58cc/609f067d-f2a7-447d-97d6-72376c287bcb)

![image.png](https://images.zenhubusercontent.com/5c6c4b4e3ba63a679b1c58cc/b4f00ff7-5c52-413d-97bc-1fe8571b1eb0)

**Comment by MargaritaRamirez** [comment]
@kenjitm 
When you land in the reporting tab you see the information from progress and after a couple of seconds the information charge with the editable fields. 
We can schedule a call because I don't know if I explaining myself

**Comment by MargaritaRamirez** [comment]
@htobon this is ready for you to test.

**Comment by htobon** [comment]
Thanks @MargaritaRamirez , can you please schedule a 30mins session where we can do this together?
Thank you in advance.

**Comment by htobon** [comment]
@MargaritaRamirez and @kenjitm something is not working well with our Github issues. I just figured out Margarita's message was written on Sep 17, 2021. Why this issue has been in review for so long?
@RodrigoOrtizM, This cannot happen! may you please work with @kenjitm and make sure this issue list is up to date?

Thank you all.
